components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2139,6 +2139,48 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
 
+  '/applications/meta/groups':
+    get:
+      tags:
+        - Application Metadata
+      summary: |
+        Retrieve the list of groups available for the application.
+      description: >
+        This API provides the capability to retrieve the list of groups available for the application. <br>
+          <b>Permission required:</b> <br>
+              * /permission/admin/manage/identity/applicationmgt/view <br>
+          <b>Scope required:</b> <br>
+              * internal_application_mgt_view
+      parameters:
+        - $ref: '#/components/parameters/domainQueryParam'
+        - $ref: '#/components/parameters/filterQueryParam'
+      operationId: getGroups
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/GroupBasicInfo'
+        '401':
+          description: Unauthorized
```

Comment:
add Error json object for the error codes

Reply:
addressed by https://github.com/wso2/identity-api-server/pull/773/commits/c1bff3f468e0412d08ce7b8c8b97d6c642d72374